tests/test_checkout.py

```diff
@@ -132,7 +132,9 @@ def outs_info(self, stage):
         FileInfo = collections.namedtuple("FileInfo", "path inode")
 
         paths = [
-            path for output in stage.outs for path in walk_files(output.path)
+            path
+            for output in stage["outs"]
```

Comment:
why do you prefer brackets over dot notation, @pared ? (just curious :stuck_out_tongue: )

Reply:
I do not, however, there was a problem with test using this functionality.
1. TestCheckoutSelectiveRemove was supposed to use cahe type copy
2. Because of mixing cli calls with api calls the config was not reloaded and test used hardlink (at least on my computer)
3. To enforce config reloading i changed self.dvc.add(self.DATA_DIR) -> main(["add", self.DATA_DIR])
4. Because of that, we do not retrieve Stage objects as a result of self.dvc.add
5. To fix that I load stage file, but it is loaded as dict, hence bracket notation.

Reply:
Got it, nice workaround, tho :stuck_out_tongue: